src/components/DatabaseList/DatabaseList.module.css

```diff
@@ -94,4 +94,5 @@
   text-align: center;
   justify-content: center;
   align-items: center;
+  padding-top: 15%;
```

Comment:
do u mind giving more context on this

Reply:
@kawerewagaba That was wrong I have removed it.